templates/configure.mk

```diff
@@ -26,12 +27,10 @@ COMPONENT_ARCHIVE_URL=
 COMPONENT_ARCHIVE_HASH=
 COMPONENT_LICENSE=
 
-include $(WS_MAKE_RULES)/prep.mk
-include $(WS_MAKE_RULES)/configure.mk
-include $(WS_MAKE_RULES)/ips.mk
+TEST_TARGET=$(NO_TESTS) # if no testsuite enabled
```

Comment:
Again: Set only if not yet done?

Reply:
Tests are enabled by default for `configure` recipes.